blocks/tabs/tabs.css

```
@@ -1,58 +1,174 @@
 .tabs .tabs-list {
-    display: flex;
-    gap: 0.5ch;
+  display: flex;
+  gap: 0.5ch;
+  max-width: 100%;
+  font-size: var(--body-font-size-xs);
+  overflow-x: auto;
+}
+
+@media (width >= 600px) {
+  .tabs .tabs-list {
+    font-size: var(--body-font-size-s);
+  }
+}
+
+@media (width >= 900px) {
+  .tabs .tabs-list {
+    font-size: var(--body-font-size-m);
+  }
+}
+
+.tabs .tabs-list button {
```

```suggestion
.tabs .tabs-list .tabs-tab {
```
Since you are using ".tabs-tab" for all other styles, you can change all  instances of "button" to ".tabs-tab" for consistency. This will make it much easier to override in a spreadsheet.